http/1.6/modules/ROOT/pages/http-request-ref.adoc

```diff
@@ -211,48 +455,77 @@ In the *Configuration XML* editor, the `requestStreamingMode` configuration look
+
+The following example shows how to configure the *Request streaming mode* and *Response buffer size* fields in Studio:
+
+. In the *Mule Palette* view, select *HTTP > Request*.
+. Drag *Request* to the Studio canvas.
+. Set *Path* to `/path`.
+. Click the plus sign (*+*) to open the HTTP Request global configuration.
+. Select *Stream response*.
+. Set *Response buffer size* to `100`.
 
-When requesting a large payload, you can choose to stream the response by setting the `streamResponse` attributes in the HTTP request configuration.
+image::http-request-16.png[HTTP Request configuration window with Stream response field selected and Response buffer size field set to 100]
 
-By default, the `streamResponse` attribute is set to false. When setting this attribute to true, you enable Mule to handle the response as a stream by saving chunks to an in-memory buffer. You can configure the buffer size using the `responseBufferSize` attribute, but its default size is 10 KB.
+In the *Configuration XML* editor, the `streamResponse` and `responseBufferSize` configurations look like these:
 
 [source,xml,linenums]
 ----
 <http:request-config name="requestConfig">
-    <http:request-connection host="localhost" port="8081" streamResponse="true" responseBufferSize="1024"/>
+    <http:request-connection host="localhost" port="8081" streamResponse="true" responseBufferSize="100"/>
 </http:request-config>
 ----
 
-To avoid issues, it's important to consume responses when streaming.
-
-== Retry Mechanisms
+== Configure Retry Mechanisms
 
-The HTTP Connector uses a retry mechanism that enables you to configure how many times it tries consuming an external HTTP service. The HTTP Connector uses this mechanism to reconnect an HTTP client with an HTTP service. The HTTP Connector does not manage TCP connections, so this mechanism does not reconnect sockets.
+HTTP Connector uses a retry mechanism that enables you to configure how many times it tries consuming an external HTTP service. The connector uses this mechanism to reconnect an HTTP client with an HTTP service. HTTP Connector does not manage TCP connections, so this mechanism does not reconnect sockets.
 
 [WARNING]
-The *HTTP Request* operation does not use the reconnection strategy for retries. The reconnection strategy is a mechanism the Mule SDK uses to re-establish _Connections_ when a `ConnectionException` is caught. It does not affect connections to an HTTP service created by the HTTP Requester.
-To learn more about _Connections_, see xref:mule-sdk::connections.adoc[Connections documentation]
+The HTTP *Request* operation does not use the reconnection strategy for retries. The reconnection strategy is a mechanism that Mule SDK uses to re-establish connections when a `ConnectionException` is caught. This mechanism does not affect connections to an HTTP service created by the HTTP *Request* operation.
```

Comment:
I kind of feel like this clarification (reconnect vs. retries) should come first, as an introduction.